netman/core/objects/vlan.py

```diff
@@ -26,3 +26,7 @@ def __init__(self, number=None, name=None, ips=None, vrrp_groups=None, vrf_forwa
         self.ips = ips or []
         self.vrrp_groups = vrrp_groups or []
         self.dhcp_relay_servers = dhcp_relay_servers or []
+        if icmp_redirects is not None:
```

Comment:
This is an interesting point.  The hardware we know have an icmp redirects state defaulting to True.  But that might not always be the case...

The adapter which instantiate the vlan is responsible to know that, but it should now leave this empty, even if instantiated empty for ease of code.

Therefore i recommend not having a default value here, let the adapter set a value, the compliance test will ensure it does